### ISSUE TITLE
iWatch size

### ISSUE THREAD

**Issue by ddodge40** [jp3141/AppleWatchAmmeter#1]
I printed the STL file and it is smaller than my iWatch.  I have a model A2774 that is larger.  Can yo make a STL file for the larger iWatch?
Thanks

**Comment by ddodge40** [comment]
Using the Apple specs page I was able with Cura to update the dimensions and 3D printed the Iwatch 8 part.
Now the Sense-app shows no data other that location.  All others are default to zero.

**Comment by jp3141** [comment]
Does your Compass app work ? Is there some permissions/privacy setting that the Sense-app needs ?

**Comment by ddodge40** [comment]
Yes compass works and sensor-app shows location. But all others are zero on both iPhone and watch Thanks


Sent from the all new AOL app for iOS


On Friday, September 27, 2024, 6:18 PM, jp3141 ***@***.***> wrote:




Does your Compass app work ? Is there some permissions/privacy setting that the Sense-app needs ?

**Comment by jp3141** [comment]
Try the Phyphox app from Aachen..., but the magnetometer should work in iWatch 8

**Comment by ddodge40** [comment]
Found phyphox for iPad , but not for watch Will try the coil with my phone.  I searched the security settings but found nothing for sensors Don DodgeCell 714-271-7888W6SNXOn Sep 27, 2024, at 7:19 PM, jp3141 ***@***.***> wrote:﻿
Try the Phyphox app from Aachen..., but the magnetometer should work in iWatch 8

**Comment by jp3141** [comment]
take it into the Apple Store and ask
